ReactiveTask/Task.swift

```diff
@@ -215,7 +215,7 @@ private final class Pipe {
 					let bytes = UnsafeMutablePointer<UInt8>(allocatingCapacity: data.count)
 					data.copyBytes(to: bytes, count: data.count)
 					let buffer = UnsafeBufferPointer(start: bytes, count: data.count)
-					let dispatchData = DispatchData(bytes: buffer)
+					let dispatchData = DispatchData(bytesNoCopy: buffer, deallocator: .custom(nil, {}))
```

Comment:
This fixes the crashing test, but it feels to me like a non-ideal thing to have to do. I mean it seems to work, I guess because `dispatchData` only gets used while `data` (which owns the bytes) exists.

It would be preferable to create `dispatchData` by copying the bytes in `data`, but doing so causes an `EXC_BAD_ACCESS` when `dispatchData` gets deallocated.